.vib/gitlab-runner/goss/vars.yaml

```diff
@@ -0,0 +1,11 @@
+binaries:
+  - gitlab-runner
+files:
+  - paths:
+      - /opt/bitnami/gitlab-runner/bin/gitlab-runner
+directories:
+  - mode: "0755"
+    paths:
+      - /opt/bitnami/gitlab-runner
```

Comment:
I think we don't really have specific logic regarding this path right? We do create the `...gitlab-runner/bin` one, but this one is not modified in any form. Even if it is the base path I don't think there is any need to check it here.